LocalSettings.php

```diff
@@ -2860,6 +2860,23 @@
 				'edittemplateprotected' => true,
 			],
 		],
+		'+nycsubwaywiki' => [
+			'steward' => [
+				'managewiki-restricted' => true,
+				'interwiki' => true,
+				'siteadmin' => true,
```

Comment:
No group, local or global, has or needs `siteadmin`, as that's all handled via LocalSettings.php by SRE.